app/components/DocumentMeta.js

```diff
@@ -103,6 +104,17 @@ function DocumentMeta({
   const collection = collections.get(document.collectionId);
   const updatedByMe = auth.user && auth.user.id === updatedBy.id;
 
+  const timeSinceNow = () => {
+    if (!lastViewedAt)
+      return <Modified highlight={true}>Never Viewed</Modified>;
```

Comment:
```suggestion
      return <Modified highlight={true}>Never viewed</Modified>;
```

The color difference on this one is currently too extreme, in light view it's very dark and stands out as much as the document title.